search-enterprise/latest/config/transactional.md

```diff
@@ -0,0 +1,301 @@
+
+TMDQ supports:
+
+* `query`
+* `paging`
+* `include`
+* `includeRequest`
+* `fields`
+* `sort`
+* `defaults`
+* `filterQueries`
+* `scope` (single)
+* `limits` for permission evaluation
+
+The default limits for permission evaluation will restrict the results returned from TMDQ based on both the number of results processed and time taken. These can be increased, if required.
+
+The v1 REST API does not support TMDQ for:
+
+* `templates`
+* `localisation` and `timezone`
+* `facetQueries`
+* `facetFields`
+* `highlight`
+* `ranges facets`
+
+The use of these with TMDQ is undefined. Some of these options will be ignored and results will come from the database; others will cause the database query to fail and ACS will fail over to return results from the search index.
+
+The V1 REST API ignores the SQL `SELECT` part of a CMIS query and generates the results as it would do for AFTS.
+
+### CMIS QL & TMDQ
+
+For CMIS QL, all expressions except for `CONTAINS()`, `SCORE()`, and `IN_TREE()` can now be executed against the database. Most data types are supported except for the CMIS uri and html types. Strings are supported but only if they are 1024 characters or less in length.
+
+Primary and secondary types are supported and require inner joins to link them together. You can skip joins to secondary types from the fetch in CMIS using the v1 REST API. You would need an explicit `SELECT` list and supporting joins from a CMIS client. You still need joins to secondary types for predicates and ordering. As CMIS SQL supports ordering as part of the query language, you have to do it there and not via the v1 REST API sort.
+
+For multi-valued properties, CMIS QL supports `ANY` semantics from SQL 92. A query against a multi-lingual property, such as title or description, is treated as multi-valued and may match in any language. In the results, you will see the best value for your locale, which may not match the query. Also, ordering will consider any value.
+
+### UPPER() and LOWER()
+
+`UPPER()` and `LOWER()` functions were in early drafts for the CMIS 1.0 specification, but were subsequently dropped. These are not part of the CMIS 1.0 or 1.1 specifications. They are not supported in TMDQ.
+
+### Alfresco FTS QL & TMDQ
+
+It is more difficult to write AFTS queries that use TMDQ as the default behaviour is to use full text queries for text and full text queries cannot be served by the database. Also, special fields like `SITE` and `TAG` that are derived from paths or other nodes will not be handled by the database. `TYPE`, `ASPECT` and the related exact matches will work with TMDQ. All property data types are fine but strings should be less than 1024 characters in length. Text queries have to be prefixed with `=` to avoid full text search.
+
+Ranges, PATH, and ANCESTOR are not currently supported.
```

Comment:
Does Ranges need to be capitalized similar to PATH and ANCESTOR?

Reply:
Cheers its supposed to be like that